rust/cadeau/src/xmf/vpx/mod.rs

```diff
@@ -139,57 +139,60 @@ pub struct VpxFrame {
 
 impl Drop for VpxFrame {
     fn drop(&mut self) {
-        // SAFETY: see SAFETY Note in VpxFrame::new, this is safe to call.
+        // SAFETY: see SAFETY Note in VpxFrame::from_raw, this is safe to call.
```

Comment:
This wording may be more appropriate.
```rust
// SAFETY: The VpxFrame type guarantees that `ptr` is a valid pointer.
```

- "Safety note" was moved out of the documentation of `VpxFrame::from_raw`
- Generally speaking, "Safety note" is not commonly used for denoting something we can rely on when proving internal unsafe preconditions. You can use this if you want to give an overview of how we ensure memory safety, but what actually documents invariants on which we rely on and which are caller responsibility are  # Safety sections.
- It’s simpler to think in term of invariants: `ptr` is a valid pointer, and we maintain this invariant at any time, either by documenting that the caller is responsible for that, or by enforcing it ourselves in some other way. In this case, the caller of `from_raw` is responsible of ensuring that this invariant is not violated. Then we can rely on this invariant.

Reply:
updated!